client/src/components/FoodSeeker/MarkerHelpers.js

```diff
@@ -90,6 +90,7 @@ export const markersLayerStyles = {
     "icon-image": ["get", "iconId"],
     "icon-allow-overlap": true,
     "icon-size": 1 / MARKER_SCALE,
+    "icon-anchor": "bottom",
```

Comment:
I noticed that the icons were centered over the point of interest, when the icon shape suggests that the actual location should be at the bottom-center of the icon (the tip of the pin), We had this problem before, and fixed it by adding pixel-based offsets to align the icon correctly, but this is much cleaner!